Patch to set the soversion be what we currently deliver

--- libXi-1.8.4/src/Makefile.am
+++ libXi-1.8.4/src/Makefile.am
@@ -75,7 +75,7 @@ $(MALLOC_ZERO_CFLAGS) \
             $(CWARNFLAGS)
 
 
-libXi_la_LDFLAGS = -version-number 6:1:0 -no-undefined
+libXi_la_LDFLAGS = -version-number 5:0:0 -no-undefined
 
 libXiincludedir = $(includedir)/X11/extensions
 libXiinclude_HEADERS = $(top_srcdir)/include/X11/extensions/XInput.h \
